Big-Spender/readme.md

```diff
@@ -68,39 +68,39 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends where lower(description) like '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends s inner join expense_areas e on (s.expense_area_id = e.id) where expense_area='Better Hospital Food';
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+select to_char(date, 'YYYY-MM') as month, sum(amount) as total from spends group by month;
```

Comment:
how else could you use to_char? Rather than `YYYY-MM`, have you thought about just using MONTH?